lucid/main_window.py

```diff
@@ -191,6 +191,8 @@ def gather_windows(self):
     @QtCore.Slot(tuple)
     def handle_error(self, data):
         exc_type, exc_value, exc_trace = data
+        if exc_value is None:
+            return
         logger.exception("An uncaught exception happened: %s", exc_value,
                          exc_info=exc_value)
```

Comment:
I think this should be the tuple here?

https://docs.python.org/3/library/logging.html
`exc_info – An exception tuple with the current exception information, or None if no exception information is available.`